object.go

```diff
@@ -34,9 +34,27 @@ func (c Color) MarshalText() ([]byte, error) {
 	return []byte(c), nil
 }
 
+type DatabaseMention struct {
+	ID ObjectID `json:"id"`
+}
+
+type PageMention struct {
+	ID ObjectID `json:"id"`
+}
+
+type Mention struct {
+	Type            ObjectType       `json:"type,omitempty"`
+	DatabaseMention *DatabaseMention `json:"database,omitempty"`
+	PageMention     *PageMention     `json:"page,omitempty"`
+	User            *User            `json:"user,omitempty"`
+	Date            *DateObject      `json:"date,omitempty"`
+}
```

Comment:
What about [template mention](https://developers.notion.com/reference/rich-text#template-mentions)?
Is it possible to add it also in this PR?

Reply:
Done